### PR TITLE
chore: package update

### PR DESCRIPTION
**Pull request by dzbo**

### CODE REVIEW

**Review by richtera — APPROVED** [review]

File: nuxt.config.ts [hunk 1]
```diff
@@ -69,7 +69,7 @@ export default defineNuxtConfig({
       alias: {
         process: 'process/browser',
         stream: 'stream-browserify',
-        https: 'agent-base',
+        https: 'stream-http',
```
Comment (Hugoo) [inline nuxt.config.ts:72]: What does this mean? There is another lib that needs to be used now? (I'm not super familiar with Nuxt config - it caught my curiosity)

**Review by Hugoo — APPROVED** [review]
Nice!

The ci break I think it is a timeout so maybe a restart could fix it. There was also this msg:

[WebServer] [warn] Plugin `/Users/runner/work/wallet.universalprofile.cloud/wallet.universalprofile.cloud/plugins/1.isBrave.ts` is not wrapped in `defineNuxtPlugin`. It is advised to wrap your plugins as in the future this may enable enhancements.

File: nuxt.config.ts (hunk 1 above)
Comment (dzbo) [reply nuxt.config.ts:72]: I was just wrong one we used before. But yes this is for Nuxt

**Comment by dzbo** [comment]
Looks like tests stuck for some reason, I don't have time to investigate now. I will make attempt to upgrade some other time.

**Pull request closed without merging**